src/patternfly/layouts/Flex/flex.scss

```diff
@@ -119,36 +118,25 @@ $pf-flex-spacer-map: (
 
   // Establish default item spacing
   > * {
-    margin-left: 0;
-
-    &:not(:last-child) {
-      margin-right: var(--pf-l-flex--spacer);
-    }
-  }
-
-  // Spacing for flex-end justification - attr selector to target base + responsive classes
-  &[class*="pf-m-justify-flex-end"] {
-    > * {
-      margin-right: 0;
-
-      &:not(:first-child) {
-        margin-left: var(--pf-l-flex--spacer);
-      }
-    }
-  }
+    margin-right: var(--pf-l-flex--spacer);
 
-  // Remove spacers for space-between
-  &[class*="pf-m-justify-space-between"] {
-    > * {
+    &:last-child {
       --pf-l-flex--spacer: 0;
     }
   }
 
   // Reset item spacing in nested flex layouts
   & & {
+    --pf-l-flex--spacer: var(--pf-l-flex--flex--spacer);
```

Comment:
Changing this default would be a breaking change. It's safer if you were to reduce the spacer, but increasing the spacer means lines may now break when they wouldn't have broken before. Can we retain the previous value?